pkgs/stdenv/generic/setup.sh

```diff
@@ -409,15 +410,14 @@ findInputs() {
     # The current package's host and target offset together
     # provide a <=-preserving homomorphism from the relative
     # offsets to current offset
-    local -i mapOffsetResult
     function mapOffset() {
         local -r inputOffset="$1"
+        local -n outputOffset="$2"
```

Comment:
Why did this change fro -r to -n?

Reply:
So before the change the code defines a variable in the outer scope, assigns to that, and then copies the result into another variable.
I remove the outer scope variable completely and use a named reference to assign to the target variable directly.
I think it makes the intent clearer since it removes one layer of indirection. Let me know if you feel strongly against it.